.github/workflows/canary_checks.yml

```diff
@@ -30,6 +30,7 @@ jobs:
           npm run test
   live_dependency_health_checks:
     runs-on: ubuntu-latest
+    timeout-minutes: 15
```

Comment:
this should be plenty enough but not too agressive
![image](https://github.com/aws-amplify/amplify-backend/assets/5849952/da00ec68-bca2-40b3-b53e-45a86fb5b495)